static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -158,6 +119,46 @@ const BreadcrumbContainer = styled('div')`
   overflow: hidden;
   border: 1px solid ${p => p.theme.border};
   border-radius: ${p => p.theme.borderRadius};
+
+  .beforeHoverTime + .afterHoverTime:before {
+    background-color: ${p => p.theme.surface200};
+    border-top: 1px solid ${p => p.theme.purple200};
+    content: '';
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 999999999%;
```

Comment:
all this css is really similar to the network and error tabs. there's some differences which is why it's not 100% shared. we could revisit and bring them closer together i suspect